NTupleMaker/interface/leptau_jets_WIP.h

```diff
@@ -222,72 +236,75 @@ void counting_jets(const AC1B *analysisTree, Synch17Tree *otree, const Config *c
     }
 
     if (jetPt > ptLeadingJet) {
+      indexSubLeadingJet = indexLeadingJet;
+      ptSubLeadingJet = ptLeadingJet;
       indexLeadingJet = jet;
       ptLeadingJet = jetPt;
     }
 
-    if (jetPt < cfg->get<float>("JetPtHighCut")) continue;
+    if (jetPt < JetPtHighCut) continue;
     jets.push_back(jet);
-  }
+  } // jets loop
 
   otree->njets = jets.size();
   otree->njetspt20 = jetspt20.size();
   otree->nbtag = bjets.size();
-  
-  otree->bpt_1   = -10; //Merijn 2019 7 31: put to -10 since it is used in the DNN don't want to confuse with strange value
-  otree->beta_1  = -9999;
-  otree->bphi_1  = -9999;
-  otree->bcsv_1  = -9999;
-  
+
+  // leading b-jet variables
   if (indexLeadingBJet >= 0) {
     otree->bpt_1   = get_jetPt(analysisTree, indexLeadingBJet, JESname, direction, jecUncertainties);
     otree->beta_1  = analysisTree->pfjet_eta[indexLeadingBJet];
     otree->bphi_1  = analysisTree->pfjet_phi[indexLeadingBJet];
-  
-    otree->bcsv_1  = analysisTree->pfjet_btag[indexLeadingBJet][0];
+    otree->bcsv_1  = analysisTree->pfjet_btag[indexLeadingBJet][3]; // 3 - pfDeepFlavourJetTags:probb
+  }
+  else {
+    otree->bpt_1   = -10;
+    otree->beta_1  = -9999;
+    otree->bphi_1  = -9999;
+    otree->bcsv_1  = -9999;
```

Comment:
Here you can put -10 to all values, it's better to have the initialization values consistent with each other